docs/features/LOGGING.md

```diff
@@ -0,0 +1,58 @@
+# Logging
+
+## Configure logging system and/or event logs to file
+By default, Crawler will output system logs to `stdout`. You have the option to write these logs to file, in addition to stdout, by setting the following in your Crawler YAML config:
```

Comment:
One last request (sorry didn't notice before): can you include that these logging configurations go in the crawler YAML file and not the filebeat YAML file?